src/components/claiming/UploadFile.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div>
+    <NavigationCard
+      data-test="header-2"
+      class="wallet-disclaimer"
+      title="Import claiming file"
+      :previousStep="previousStep"
+      :nextStep="nextStep"
+      previousText="Back"
+      nextText="Next"
+    >
+      <InformativeContent :subtitle="subtitle" :texts="texts" />
+      <el-upload
```

Comment:
If you go to the next step and then go back, you have lost the uploaded file. Can you create an issue to keep the file uploaded?